src/client/unittests/display/picker.ts

```diff
@@ -55,7 +55,7 @@ export class TestDisplay {
                 testFunctions.some(testFunc => testFunc.nameToRun === fn.testFunction.nameToRun);
         });
 
-        window.showQuickPick(buildItemsForFunctions(rootDirectory, flattenedFunctions),
+        window.showQuickPick(buildItemsForFunctions(rootDirectory, flattenedFunctions, undefined, undefined, debug),
```

Comment:
It is worth providing default parameters for `buildItemsForFunctions()`?

Reply:
The ```buildItemsForFunctions``` already has default values defined for the parameters. I.e. passing ```undefined``` will result in the those parameters defaulting to their default values.

https://github.com/DonJayamanne/pythonVSCode/pull/1299/files/1d0fdcb401c0476e7f9181bd7d57c4aeb257d557#diff-e4c557dad6f86448a80a2c1a7b2fa045R134